Form-Controls/index.html

```diff
@@ -13,16 +13,46 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <form action="results.html" method="get">
+    <div>
+      <label for="name">Name <span>*</span></label>
+      <input type="text" placeholder="Your Name" name="Name" id="name" pattern="^[A-Za-zÀ-ÖØ-öø-ÿ' -]+$" minlength="2" required>
```

Comment:
You did great job! Could you check your [form](https://deploy-preview-299--cyf-ufd.netlify.app/form-controls/) please ? It seems pattern is not working!

Reply:
Pattern is removed in [c31c820](https://github.com/CodeYourFuture/Module-User-Focused-Data/pull/299/commits/c31c820049af4ed4a790a8948e1162dc9b1ec3fe)